### PR TITLE
Add in-depth description to custom abilities. Add notice of the innate ability above said innate

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: server/chat-plugins/random-battles.ts [hunk 1]
```diff
@@ -603,9 +606,11 @@ function SSBSets(target: string) {
 			buf += generateSSBPokemonInfo(item.megaStone, dex, baseDex);
 		} else if (set.species === 'Rayquaza') {
 			buf += generateSSBPokemonInfo('Rayquaza-Mega', dex, baseDex);
-		// Struchni has itemless Mega Evolution
+		// Struchni and Raj.shoot has itemless Mega Evolution
```
Comment (AnnikaCodes) [inline server/chat-plugins/random-battles.ts]:
```suggestion
		// Struchni and Raj.shoot have itemless Mega Evolution
```

Comment (AnnikaCodes) [reply server/chat-plugins/random-battles.ts]: this whole thing can be refactored
```ts
} else if (['Aggron', 'Charizard', 'Rayquaza'].includes(set.species)) {
    buf += generateSSBPokemonInfo(`${set.species}-Mega`, dex, baseDex);
}
```

**Review by AnnikaCodes — CHANGES_REQUESTED** [review]

File: server/chat-plugins/random-battles.ts [hunk 2]
```diff
@@ -439,6 +439,9 @@ function generateSSBAbilityInfo(set: SSBSet, dex: ModdedDex, baseDex: ModdedDex)
 			if (details[detail] === '') return detail;
 			return `<font color="#686868">${detail}:</font> ${details[detail]}`;
 		}).join("&nbsp;|&ThickSpace;")}</font>`;
+    if (sigAbil.desc) {
+  		buf += `<details><summary><strong>In-Depth Description</strong></summary>${sigAbil.desc}</details>`;
+  	}
```
Comment (AnnikaCodes) [inline server/chat-plugins/random-battles.ts]:
```suggestion
		if (sigAbil.desc) {
			buf += `<details><summary><strong>In-Depth Description</strong></summary>${sigAbil.desc}</details>`;
  	}
```

Comment (ghost) [reply server/chat-plugins/random-battles.ts]: ty

File: server/chat-plugins/random-battles.ts (hunk 1 above)
Comment (ghost) [reply server/chat-plugins/random-battles.ts]: coolty

**Pull request merged**